src/types/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './aggregation'
+export * from './aggregation/aggregation'
```

Comment:
Since you are only exporting from here, you should make the file `./aggregation/index.ts` so you can make this `export * from './aggregation/`

Reply:
Done